examples/reinforcement_learning_controllers/ddpg_series_omega_control.py

```diff
@@ -49,8 +49,8 @@
         reward_power=0.5,
 
         # Define which state variables are to be monitored concerning limit violations
-        # None means that a limit violation will never trigger an env.reset
-        observed_states=None,
+        # () means that a limit violation will never trigger an env.reset
+        constraints=(),
```

Comment:
I like that we are avoiding the term "observed states" in this way, as it had some error potential in the context of control theory.